.github/styles/Vocab/FluxNinja/accept.txt

```diff
@@ -81,3 +81,4 @@ write_http
 Kubelet
 colocating
 [Oo]perator
+withH
```

Comment:
Remove from here and add to pre commit config

Reply:
Better to just exclude the file from codespell scan

Reply:
ok